target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -78,6 +78,7 @@ ramips_setup_interfaces()
 	dir-320-b1|\
 	dir-610-a1|\
 	dir-615-h1|\
+	gb-pc1|\
```

Comment:
RJ45 jacks are connected to 0 and 4 ...

I pushed this :

	zbt-wa05)
		ucidef_add_switch "switch0" \
			"1:lan" "2:lan" "3:lan" "4:lan" "0:wan" "6@eth0"
		;;
	gb-pc1)
		ucidef_add_switch "switch0" \
			"0:lan" "4:lan" "6@eth0"
		;;
	gl-mt300n-v2)
		ucidef_add_switch "switch0" \
			"1:lan" "0:wan" "6@eth0"
		;;